setup.py

```diff
@@ -4,7 +4,7 @@
 import versioneer
 import sys
 
-README = open('README.rst', 'r').read()
+README = open('README.md', 'r').read()
```

Comment:
`long_description_content_type='text/markdown'` needs to be added in the setup call so that it renders correctly on PyPI.